triteo_linear.py

```
@@ -0,0 +1,128 @@
+            S_temp = T.alloc_fragment(d, 'float32') 
+            S_down = T.alloc_fragment(d, 'float32') 
+            S_up   = T.alloc_fragment(d, 'float32') 
+            S_mid  = T.alloc_fragment(d, 'float32') 
+            for i0_d in T.Parallel(d):
+                S_temp[i0_d] = 0 
+                S_down[i0_d] = 0
+                S_up[i0_d]   = 0
+                S_mid[i0_d]   = 0
+            for i0_t in T.serial(blk_t):
+                t_local = i0_t*blk_t + i0_t
+                #先存第一行也就是栈顶，到输出的o里面
```

Ruff (0.14.4)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)